formatTeams.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import csv
+import random 
+
+if len(sys.argv) != 4: 
+	print("Invalid amount of arguments: Should be maxChar maxInt")
+	sys.exit()
+
+maxChar = ord(sys.argv[2])
+maxInt = int(sys.argv[3])
+
+unformatted = open(sys.argv[1], "r", encoding="utf8")
+formatted = open("sorted.csv", "w", encoding="utf8")
+
+csv_reader = csv.DictReader(unformatted, delimiter=',')
+
+# create array maxChar(maxInt) size randomized 
+array = []
+for i in range(ord('A'), maxChar + 1):
+	for j in range(0, maxInt):
+		table = ("%s%i" % (chr(i), j))
+		array.append(table)
+random.shuffle(array)
+
+# header row in new csv
+formatted.write('Team,Table location,DevPos Url,Desired Prizes\n')
+
+x = 0
+for lines in csv_reader:
```

Comment:
Does `x` get incremented anywhere? It looks like what you would want to use instead is `enumerate`, since you want the index of the current line. 
```suggestion
for x, lines in enumerate(csv_reader):
```
That way, you can get away with not having to define `x` and keeping track of incrementing it. It'll also be a good practice to check if x will go out of index in the array, and it does, it should print an error informing the user to input a larger maxChar and/or maxInt and exit.